measures/qoi_report/measure.rb

```diff
@@ -456,6 +497,35 @@ def daily_peak_timing_stats_by_month(timeseries, month_val, stats_option = 'med'
     stats_by_month
   end
 
+  def grid_peak_schedule(model, peak_len)
+    grid_region = model.getBuilding.additionalProperties.getFeatureAsString('grid_region')
+    raise 'Unable to find grid region in model building additional properties' unless grid_region.is_initialized
+    grid_region = grid_region.get
+    load_csv = "#{File.dirname(__FILE__)}/cambium/#{scenario}/#{grid_region}.csv"
+    return nil if !File.file?(load_csv)
+    net_load_mwh = CSV.read(load_csv, converters: :float).flatten
+    grid_peak_schedule = peak_schedule_generation(annual_load = net_load_mwh, oat = timeseries['temperature'], peak_len = peak_len, num_timesteps_in_hr = 1, peak_window_strategy = 'center with peak', rebound_len = 0, prepeak_len = 0, season = 'all')
+    grid_peak_schedule
+  end
```

Comment:
maybe I'm just not so experienced with coding, but is it safe to not include `timeseries` as input arg for this method?